src/analyzer/plugins/analyzersoundtouchbeats.cpp

```diff
@@ -8,15 +8,14 @@
 namespace mixxx {
 
 AnalyzerSoundTouchBeats::AnalyzerSoundTouchBeats()
-        : m_downmixBuffer(kAnalysisFramesPerChunk), // mono, i.e. 1 sample per frame
-          m_fResultBpm(0.0f) {
+        : m_downmixBuffer(kAnalysisFramesPerChunk) {
 }
```

Comment:
won't this cause `m_resultBpm` to be uninitialized in some cases?

Reply:
It's should be default-constructed, i.e. invalid.

Reply:
Ok, I'm not familiar with the rules on this in C++. Someone else should judge this.